flask_rest_orm/resources/utils.py

```diff
@@ -1,5 +1,31 @@
 from sqlalchemy import desc
+import json
+import operator
 
+OPERATORS = {
+    'like': 'like',
+    'notlike': 'notlike',
+    'ilike': 'ilike',
+    'notilike': 'notilike',
+    'is': 'is_',
+    'isnot': 'isnot',
+    'match': 'match',
+    'startswith': 'startswith',
+    'endswith': 'endswith',
+    'contains': 'contains',
+    'in': 'in_',
+    'notin': 'notin_',
+    'between': 'between',
+}
+
+OPERATE = {
```

Comment:
A better name would be `COMPARE_OPERATORS`. Would be also good to have some docs on why you did separate them in two lists

Reply:
OK